cmd/sidecars/network-passt-binding/BUILD.bazel

```diff
@@ -18,6 +18,7 @@ go_library(
 go_binary(
     name = "network-passt-binding",
     embed = [":go_default_library"],
+    static = "on",
```

Comment:
This binary is run in a container, and I would assume it should use the libraries from that container. Why does it work on some platforms and fails on others? This should be independent of the host OS. Or am I missing smth?

Reply:
It is also included as a plain binary in the release assets - https://github.com/kubevirt/kubevirt/releases/tag/v1.1.1

/cc @ormergi  - is the network-passt-bindng binary meant to be included in the release assests? I guess the published artifact is meant to be the image?

Reply:
In the release assets we deliver the custom CNI [1].

https://github.com/kubevirt/kubevirt/tree/main/cmd/cniplugins

So I think @vasiliy-ul has a point here.

Reply:
Ah, okay, so we have the sidecar binary `network-passt-binding` (this one, which is run in a container) and also a CNI binary `kubevirt-passt-binding` (which is run on the host). Then `kubevirt-passt-binding` needs to be built statically, I assume:

https://github.com/kubevirt/kubevirt/blob/f4aad6e368e8a4fa54fea866939b84a3b27b5512/cmd/cniplugins/passt-binding/cmd/BUILD.bazel#L16-L20

Reply:
Posted https://github.com/kubevirt/kubevirt/pull/11653

Reply:
Thanks @EdDev !